service/models.py

```diff
@@ -20,7 +20,7 @@ class Supplier(Document):
     app = None
 
     # Table Schema
-    supplierName = StringField(required=True)
+    supplierName = StringField(required=False)
```

Comment:
I think we should keep at least one field as **required**, otherwise an empty document can be inserted to the suppliers collection...

Reply:
Done